docs/resources/index.md

```diff
@@ -1,25 +1,83 @@
 # Resources for Organizers
 
-The eScience Institute supports Community Partners in planning each hackweek. We invite organizers to join a series of planning sessions to prepare for an event. Not everyone is expected to attend every session, and your level of commitment will depend on your chosen role.
+The eScience Institute supports Community Partners in planning each hackweek. We invite organizers to participate in video and live training sessions tailored to each of the Community Partner roles. Upon completion of the training, organizer profiles will be posted on the hackweek website along with badge(s) representing their roles in the event. 
```

Comment:
Should we add a link to the hackweek website so people can see where they'll be posted?